SUPPORT.md

```diff
@@ -0,0 +1,11 @@
+# Support
+
+## How to file issues and get help  
+
+This project uses GitHub Issues to track bugs and feature requests. Please search the existing 
+issues before filing new issues to avoid duplicates.  For new issues, file your bug or 
+feature request as a new Issue.
+
+## Microsoft Support Policy  
+
+Support for this Oryx is limited to the resources listed above.
```

Comment:
Nit: this sentence sounds a bit strange (`Support for this Oryx`?). If this was auto generated, please ignore. If not, please consider updating to something slightly different.